fabtests/prov/efa/src/efa_exhaust_mr_reg_common.c

```diff
@@ -2,8 +2,7 @@
 
 #include "efa_exhaust_mr_reg_common.h"
 
-int ft_efa_setup_ibv_pd(struct ibv_pd **pd)
-{
+struct ibv_context *ft_efa_open_ibv_device() {
```

Comment:
I think the canonical signature is `int ft_efa_open_ibv_device(struct ibv_context *ctx)`

For nil-parameter functions we should do `function(void)`

Reply:
fixed, thanks!